CppCoreGuidelines.md

```diff
@@ -19031,6 +19032,28 @@ Failing to follow this results in difficult to diagnose errors for clients of a
 
 A test should verify that the header file itself compiles or that a cpp file which only includes the header file compiles.
 
+### <a name="Rs-incscope"></a>SF.12: Use the quoted or angle bracket form of `#include`  to identify the scope of dependencies
+
+##### Reason
+
+To understand dependencies it is important to know where included files come from. Given the different search algorithms, 
+the angle form (`<>`) identifies global scope, the quoted (`""`), files within the project (files co-located with use of the include statement).
```

Comment:
Is ", files within the project (in the same directory as the including source file)" better understandable?